tests/sqlparser_snowflake.rs

```diff
@@ -2483,6 +2483,34 @@ fn test_sf_trailing_commas() {
     snowflake().verified_only_select_with_canonical("SELECT 1, 2, FROM t", "SELECT 1, 2 FROM t");
 }
 
+#[test]
+fn test_sf_from_trailing_commas() {
```

Comment:
hmm are the tests relying on any snowflake specific feature, I imagined they should be part of the `test_trailing_commas_in_from` in common instead? essentially moving the tests to that file